lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -41,25 +41,27 @@ def initialize(options={})
       end
 
       def purchase(money, payment, options={})
-        post = { payment: {} }
+        post = { payment: { responsible: {}} }
```

Comment:
Let's move the addition of this sub-element to the `add_customer_responsible_person` method.